deployments/edgenodesv2.nix

```diff
@@ -1,32 +1,47 @@
-{ accessKeyId, nodes ? 1 , walletsPerNode ? 1
+{ accessKeyId, nodes ? 1 , walletsPerNode ? 10
 , region ? "eu-central-1"
 , topologyFile, systemStart }:
 
 with import ../lib.nix;
 let
-  mkNode = index: {
-    name = "edgenode-${toString index}";
-    value = {
-      imports = [ ../modules/cardano-benchmark.nix ];
-      services.cardano-benchmark.index = index;
-    };
+  iohkpkgs = import ../. {};
+  params = index: {
+    walletListen = "127.0.0.1:${toString (8090 + index)}";
+    ekgListen = "127.0.0.1:${toString (8000 + index)}";
+    environment = "override";
+    confKey = "bench";
+    inherit topologyFile;
+    extraParams = "--system-start ${toString systemStart}";
   };
-in {
-  defaults = { resources, ... }: {
-    services.cardano-benchmark = {
-      inherit systemStart walletsPerNode;
-      topologyFile = "${topologyFile}";
+  mkService = index: {
+    users.users."cardano-node-${toString index}" = {
+      isNormalUser = true;
+    };
+    systemd.services."cardano-node-${toString index}" = {
+      wantedBy = [ "multi-user.target" ];
+      serviceConfig = {
+        User = "cardano-node-${toString index}";
+        WorkingDirectory = "/home/cardano-node-${toString index}";
```

Comment:
can you revert all changes to this file relative to develop?, this has already been merged in another PR and these changes are undoing some improvements

Reply:
Done.